web-client/integration-tests/journey/docketClerkAddsAndServesDocketEntryFromOrder.js

```diff
@@ -72,110 +74,112 @@ export const docketClerkAddsAndServesDocketEntryFromOrder = (
     expect(cerebralTest.getState('form.freeText')).toEqual('Order');
     expect(cerebralTest.getState('form.serviceStamp')).toBeFalsy();
 
-    // eventCode: OCA
-    await cerebralTest.runSequence(
-      'updateCourtIssuedDocketEntryFormValueSequence',
-      {
-        key: 'eventCode',
-        value: 'OCA',
-      },
-    );
+    if (testFormBehavior) {
+      // eventCode: OCA
```

Comment:
Nit: are all the eventCode comments cleanup-worthy?

Reply:
Those do seem redundant, don't they?